README.md

```diff
@@ -1,70 +1,78 @@
+Mobile ScreenShots
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## Built With
 
-### `npm test`
+- React.
+- Redux.
+- JavaScript.
+- SpaceX api
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Video
 
-### `npm run build`
+[Link](https://www.loom.com/share/a451a571209c414b8663b7d8a2a812ba) to video on Loom.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+### Prerequisites
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+  - Read about [React](https://reactjs.org/docs/getting-started.html) before start.
+  - Just follow the steps for Getting started, Asset Managment(optional), Output Management and Development (Install the webpack-dev-server)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+### Setup
+  - Better have [VS Code](https://code.visualstudio.com/) installed or install the source-code editor of your preference. 
 
-### `npm run eject`
+### Install
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+  - In the terminal go to the folder you going to work and write the next. 
+  ```
+  git clone https://github.com/eri8-9/Space-Travelers-Hub.git
+  ```
+  - Next go to VS Code editor and open the folder where you installed the repository.
+### Usage
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+- Clone the repo.
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+- Run `npm install` on your terminal.
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+- Run `npm run build` on your terminal.
 
-## Learn More
+- Open the terminal in your machine and run `npm start` to start server.
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## Authors
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+👤 **Christian Erick Contreras**
 
-### Code Splitting
+- GitHub: [@eri8-9](https://github.com/eri8-9)
+- Linkedin: [LinkedIn](https:linkedin.com/in/christian-erick-contreras-9945b820b)
```

Comment:
- [ ] Kindly make sure to provide a valid link to your LinkedIn account 😄 🙏 This is for a completely professional README file. 💯